src/index.js

```diff
@@ -0,0 +1,107 @@
+import './style.css';
+import TodoList from './modules/todo.js';
+import toggleStatus from './modules/clear.js';
+
+let todolist = [];
+if (JSON.parse(localStorage.getItem('todolist'))) {
+  todolist = JSON.parse(localStorage.getItem('todolist')).todolist;
+}
+const newTodoList = new TodoList(todolist);
+const todoItems = document.getElementsByClassName('todolist')[0];
+
+const sortedTodoList = todolist.sort((a, b) => a.index - b.index);
+
+sortedTodoList.forEach((todo) => {
+  const task = document.createElement('li');
+  task.classList.add('task');
+  task.id = todo.index;
+  task.innerHTML = `<input type="checkbox" name="${todo.index}" class="check">
+  <label class = "${todo.index} task-desc black" for="${todo.index}">${todo.description}</label>
+  <div class="remove-button">
+    <i class='fa fa-trash ash'></i>
+  <div>`;
+  todoItems.appendChild(task);
+});
```

Comment:
- [ ] Great work updating the values for `completed` upon user actions :clap: However the input's checkbox is not reflecting the status ( value of `completed`) of the task. The checkbox should stay checked when the value for `completed` is equal to `true`. Kindly check the video below :point_down:  to get more understanding of the issue.

  **HIT: We can fix this bug by checking the status of `completed` when rendering each task. It could look like this :point_down:**
```js
  task.innerHTML = `<input type="checkbox" ${todo.completed && 'checked="true"'} name="${todo.index}" class="check">
  <label class = "${todo.index} task-desc black" for="${todo.index}">${todo.description}</label>
  <div class="remove-button">
    <i class='fa fa-trash ash'></i>
  <div>`;
```

<video src="https://github.com/yayaismaail/to-do-list/assets/91142432/6aac3163-9d01-4a3a-9479-c7955af3cfcc" alt="video" />

Reply:
Can you help me please to fix the problem, because I have been struggling to arrive at that but I couldn't.
I will appreciate if you can spare some of your time to assist me.
Thank you in anticipation.

Reply:
@yayaismaail, I provided the solution in the **HIT**. 
But sorry for not going into more detail. What I meant was that we can update the code on line 14 in your `index.js` file to look like this :point_down: More precisely line 18 by adding this code `${todo.completed && 'checked="true"'}` to the checkbox input.
```js
sortedTodoList.forEach((todo) => {
  const task = document.createElement('li');
  task.classList.add('task');
  task.id = todo.index;
  task.innerHTML = `<input type="checkbox" ${todo.completed && 'checked="true"'} name="${todo.index}" class="check">
  <label class = "${todo.index} task-desc black" for="${todo.index}">${todo.description}</label>
  <div class="remove-button">
    <i class='fa fa-trash ash'></i>
  <div>`;
  todoItems.appendChild(task);
});
```

Hope this is helpful :pray:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.